src/graphql-server/schema.graphql

```diff
@@ -47,4 +85,8 @@ type Mutation {
   enableHapp(appId: String): Happ
   disableHapp(appId: String): Happ
   updateHostPricing(units: String, pricePerUnit: String): HostPricing
+  holofuelRequest(counterparty:String, amount:Float): Transaction
+  holofuelOffer(counterparty:String, amount:Float, requestId:ID): Transaction
+  holofuelAcceptOffer(transactionId: ID): Transaction # NOTE: holofuelAcceptOffer DOES NOT return the amount or counterparty. IF need these values, query from cache, as they will not change via this API.
+  holofuelRejectOffer(transactionId: ID): Transaction # NOTE: holofuelAcceptOffer DOES NOT return the amount or counterparty. IF need these values, query from cache, as they will not change via this API.
```

Comment:
is this comment true of `holofuelRejectOffer`? if so can you update the comment to fix the mutation name. How do we handle rejecting requests? Is there a zome function for that? Or if this same mutation handles both, can we call it `holofuelRejectTransaction`

Reply:
There is an reject endpoint that specifically rejects an offered payment, not a request.  It currently takes in the same parameters as the receive_payment endpoint such that, like receive_payment, the reject endpoint in the holofuel DNA will need to be updated to only take the ID instead.

For reference, here is the current endpoint : 
```
    reject: {
            inputs: |promise: Promise, promise_sig: Signature, promise_commit: Address,
                     reason: Option<String>|,
            outputs: |result: ZomeApiResult<Address>|,
            handler: transaction::reject_handler
        }
```

I will update the comment.

Reply:
The option to cancel a request has been discussed; it just needs to be implemented.